plugins/modules/vmware_sdrs_vm_overrides_info.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Lionel Sutcliffe <sutcliffe.lionel@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: vmware_sdrs_vm_overrides_info
+short_description: Check for VM overrides in a SDRS datastore cluster
+description:
+    - This module can be used to check if VM overrides exist in a SDRS enabled datastore cluster in a given VMware environment.
+    - All parameters and VMware object values are case sensitive.
+author:
+-  Lionel Sutcliffe
+options:
+    datastore_cluster_name:
+      description:
+      - The name of the datastore cluster.
+      required: true
+      type: str
+extends_documentation_fragment:
+- community.vmware.vmware.documentation
+
+"""
+
+EXAMPLES = r"""
+- name: Check if SDRS Overrides exist in datastore cluster
+  community.vmware.vmware_sdrs_vm_overrides_info:
+    hostname: '{{ vcenter_hostname }}'
+    username: '{{ vcenter_username }}'
+    password: '{{ vcenter_password }}'
+    datastore_cluster_name: '{{ datastore_cluster_name }}'
+  delegate_to: localhost
+"""
+
+RETURN = r"""
+result:
+    description: information about datastore clusters with VMs with SDRS enabled
+    returned: always
+    type: dict
+    sample: {
+        "changed": false,
+        "result": {
+            "DSC_1": {
+                "vm_overrides": {
+                    "VM_1": {
+                        "srds_enabled_status": false, "vm_behavior": null
+                        },
+                     "VM_2": {
+                        "srds_enabled_status": false, "vm_behavior": null
+                        }
+                    }
+                }
+            }
```

Comment:
This looks wrong. I get `result` and then `vm_overrides` directly. That is, without the name of the datastore cluster (which I guess `DSC_1` stands for as an example) in between. I think you should either fix this in the documentation, or change your code accordingly so that the datastore cluster name turns up in the result like you've documented.

I'm not sure what would be better. Changing the documentation to how the result really looks like might be easier. But maybe there are reasons why it should be included. Anyway, the documentation should be correct. The rest LGTM.

Sorry for the late reply, I didn't find the time to have a closer look until today.